doc/articles/features/dialogs.md

```diff
@@ -120,7 +120,7 @@ new Button()
 
 ---
 
-If you are using Uno.Extensions Navigation, you can utilize its features to display a dialog. For more information, check out this documentation: [Dialogs with Uno.Extensions Navigation](xref:Uno.Extensions.Navigation.HowTo-ShowDialog).
+If you are using Uno.Extensions Navigation, you can utilize its features to display a dialog. For more information, check out this documentation: [Dialogs with Uno.Extensions.Navigation](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Navigation/HowTo-ShowDialog.html).
```

Comment:
Thank you for your contribution, @DevTKSS! It is preferred to use the id of the given page instead of a direct link, but you are correct, the correct link should be different. This change should work (taken from [here](https://github.com/unoplatform/uno.extensions/blob/main/doc/Learn/Navigation/HowTo-ShowDialog.md/#L1)):

```suggestion
If you are using Uno.Extensions Navigation, you can utilize its features to display a dialog. For more information, check out this documentation: [Dialogs with Uno.Extensions.Navigation](xref:Uno.Extensions.Navigation.HowToShowDialog).
```

Reply:
tryed it with the uid or what you recommended there using the github editor on forked repo and the preview was wrong or link did not work properly. I thing that comes because the documentation is on uno repo and the file the link goes to is in the uno.extensions repo, which seems at least for uno not be the same. Did you check the preview and tryed the link with the href way?

Reply:
Yeah, I think the GitHub live editor doesn't work very well with UIDs. Once we commit the suggestion, I'll run our workflow to generate a staging page link, so we can both test the changes.